packages/flutter_tools/lib/src/commands/daemon.dart

```diff
@@ -736,7 +736,7 @@ class AppDomain extends Domain {
     if (app == null) {
       throw DaemonException("app '$appId' not found");
     }
-    final FlutterDevice device = app.runner!.flutterDevices.first!;
+    final FlutterDevice device = app.runner!.flutterDevices.first;
```

Comment:
I can't comment on the right lines, but can you swap the `dynamic`s in this file for `Object` or `Object?`s?

Reply:
done.